elements/centos-linkup-extra/install.d/98-centos-linkup-extra

```diff
@@ -8,3 +8,5 @@ set -o pipefail
 
 # Add a LINKDELAY value to /etc/sysconfig/network
 echo -e "# Allow extra time for link up\nLINKDELAY=${CENTOS_LINKUP_EXTRA_SECS}" >> /etc/sysconfig/network
+# Prevent Cloudinit writing over LINKDELAY setting
+echo -e "network: {config: disabled}" >> /etc/cloud/cloud.cfg.d/99-disable-custom-net-conf.cfg
```

Comment:
This seems like quite a drastic thing to always do. Network bootstrapping is a big part of cloud-init. Would it not also break network configuration provided via config-drive?